### DISCUSSION TITLE
Character with original .xml not working at all...

### DISCUSSION THREAD

**Discussion by PSI3x** [FunkinCrew/Funkin#3926] [Q&A] [answered]
For a specification of what I mean: A character whose sprite was an edit of GF's .fla worked well, but a character who used an .fla I made from scratch I kept getting a very unspecific error. Does anyone with actual knowledge with the engine's workings know what I'm doing wrong?
![MysteriousError](https://github.com/user-attachments/assets/96c8576c-45ac-424b-bcad-ae02effaf817)
This displays every time I open the character .json+.xml in the animation editor. I can send the files for testing, if you need them.

**Comment by KoloInDaCrib** [comment]
May you please send the .png and the .xml files? I took a gander at the code and line 503 is this:
![image](https://github.com/user-attachments/assets/46b1a9e9-aab0-404b-990a-e8c296305b04)

which basically means that, if you character crashes the editor, there was a problem with generating frames from the .xml file

**Reply by PSI3x** [reply]
Interesting, thank you. Here are all the files currently relating to the character sprites in question:
[XML+PNG+JSON.zip](https://github.com/user-attachments/files/18085713/XML%2BPNG%2BJSON.zip)

**Reply by KoloInDaCrib** [reply, marked as answer]
The problem lies in this line in the .json file
![image](https://github.com/user-attachments/assets/6fccbec2-f8a1-4c66-8e75-f2390f8bc0cb)

it should be 
> "assetPath": "characters/senpai_playable",

(basically just replace the dash with an underscore, since your character's files are named `senpai_playable` and not `senpai-playable`)

**Reply by PSI3x** [reply]
I see, my bad for messing up so badly. Thank you very much for the help.
